hugegraph-llm/src/hugegraph_llm/operators/index_op/build_semantic_index.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, embedding: BaseEmbedding):
         self.vid_index = VectorIndex.from_index_file(self.index_dir)
         self.embedding = embedding
 
+    def extract_names(self, vertices: list[str]) -> list[str]:
+        return [v.split(":")[1] for v in vertices]
```

Comment:
we also support a customized string named "1:tom", so we need to check the vertex id mode before we format it